CHANGES.rst

```diff
@@ -95,6 +95,12 @@ Bugfixes
 
 - None so far :)
 
+Internal Changes
+^^^^^^^^^^^^^^^^
+
+- Add ``location_info`` block in conference display template (thanks
+  :user:`omegak`)
+
```

Comment:
Wrong version; should be listed in 2.3 (also add a reference to the PR now that you know its ID)

Reply:
not sure if this really needs a changelog entry though...

Reply:
Removed.